tests/ci/data/profiles/rosa-hcp.yaml

```diff
@@ -10,6 +10,7 @@ profiles:
     sts: true
     byo_vpc: true
     private_link: false
+    additional_principals: true
```

Comment:
According to what I know from Jeff, additional_principals should be set on a private link cluster. Can you confirm with that?